lib/cosmos/packets/packet.rb

```diff
@@ -341,7 +341,7 @@ def check_bit_offsets
       previous_item = nil
       warnings = []
       @sorted_items.each do |item|
-        if expected_next_offset and item.bit_offset < expected_next_offset
+        if expected_next_offset && item.bit_offset < expected_next_offset && !item.overlap
```

Comment:
I know you prefer &&, but I still think it is more dangerous than using "and" unless you use lots of parens.  Look at the differences in the Ruby operator precedence table.

Reply:
I'll swtich back